### DISCUSSION TITLE
Jupytor lab帳號問題

### DISCUSSION THREAD

**Discussion by EricAgumonJou** [1101-GS4719-C/Python-Community-GS4719-C#12] [General]
想詢問一下:
Jupyter lab的帳號有需要額外註冊帳號嗎?
今天在命令提示字元輸入jupyter notebook後, 並開啟網頁後發現右上角有出現"登出"的選項
不知該帳號是否為以前使用過的帳號，還是系統自動幫我登入之帳號

**Comment by chia-kai-chang** [comment]
同學你看的很細耶~ 我們先不用按Logout，如果不小心按Logout，只要回到命令提示字元重新啟動Jupyter Lab即可。

Jupyter就是一個網頁服務。我們今天是在本機瀏覽器使用本機的Jupyter網頁服務。

也可以把Jupyter網頁服務架在宿舍或實驗室的電腦A中，然後在別台電腦B **登入** 宿舍電腦A 的Jupyter服務。

![image](https://user-images.githubusercontent.com/1768891/134363629-2a705d98-448b-4689-b182-bd45df9cbf85.png)

### 如果你有一張NVIDIA 3080 Ti放在宿舍或實驗室電腦中，希望隨時在外面使用輕薄筆電B連線到實驗室電腦A使用NVIDIA 3080 Ti做高速平行計算(訓練AI模型)，才需要這麼做。

因為我們只會學習Python程式語法，所以，在本機開啟Jupyter Lab服務操作即可。

**Reply by jerry850313** [reply]
感覺跟colab有點像

**Comment by enoch-lightning** [comment]
NVIDIA 3080 Ti就是伺服器(server)嗎？所以由電腦B連到電腦A的jupyter lab，就是要使用網路連接？看起來很酷，如果有機會，想要做做看。

**Reply by chia-kai-chang** [reply]
NVIDIA 3080 Ti 是遊戲用顯示卡
裝載著N社顯卡的電腦，顯卡內有許多CUDA Core可以做平行計算，可以提高計算速度
一般我們使用的筆記型電腦沒有這麼大的算力，所以可以連到裝有N社顯卡的電腦，使用Jupyter Lab的網頁介面來做程式撰寫，計算也在效能強大的電腦上計算。筆電只是顯示計算結果與撰寫程式的功能而已。

**Reply by enoch-lightning** [reply]
所以NVIDIA 3080 Ti是一種N社顯卡嗎？而N社顯卡是可以做平行運算的遊戲用顯示卡？

**Reply by enoch-lightning** [reply]
🐰🐰🐰🐰🐰🐰🐰🐰🐰🐰🐰🐰🐰🐰

**Reply by chia-kai-chang** [reply]
沒錯

**Reply by enoch-lightning** [reply]
謝謝老師。Thank you teacher.

**Reply by enoch-lightning** [reply]
我有一個問題，遊戲用顯示卡與GPU、TPU的差別是什麼？還是它們其實都是同一個東西。因為感覺它們的功能都一樣，就是算比較快。對於算比較快，好像又可以細分。如果是跑同樣的迴圈，遊戲用顯示卡、GPU與TPU會跑的比我的筆電快嗎？遊戲顯示卡、GPU與TPU「跑比較快」是在特定的情況下嗎？
